### PR TITLE
Crud (create update remove)

### PR DESCRIPTION
**Pull request by BregornOriginal**
# To Do List Add & Remove functionality

### In this part of the project were added the functionality to add a new task and delete one.

- The before part of this project was reused to add a task and implementation function.
- Deleting a task function was added
- Editing functionality was added to the project
- When you add or delete an object the index is updated
- This functionality is updating the local storage for each function

### CODE REVIEW

**Review by Gambit142 — CHANGES_REQUESTED** [review]
Hi @BregornOriginal 👋, 

Good job so far!

## Highlights
- All hardcoded items from the tasks array are removed 💯 
- A function for adding tasks was implemented 💯 
- A function for editing tasks was implemented 💯
- A function for removing a single task was implemented 💯

However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
- Based on one of the project's requirements, kindly move the functions for adding, editing and removing a single task to a new Javascript file and import them to your **_index.js_** file. 

- _Kindly check the comments under the review for other required changes._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

- In your `README.md` file, you created the _**Prerequisite**_ section twice. Kindly merge the instructions and use one _**Prerequisite**_ header. This would make your `README.md` file more professional. 👍 

image

- To install this project, you can run the following command:
  `npm install`
- Then to run the server, run `npm start`. Kindly reflect this in the install section of your `README.md` file. 👍 

image


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: src/index.js [hunk 1]
```diff
@@ -1,65 +1,122 @@
...
+});
+
+$addButton.addEventListener('click', (e) => {
+  if ($textNewTask.value) {
+    e.preventDefault();
+    addTask($textNewTask.value);
+    $textNewTask.value = '';
+    console.log(JSON.parse(localStorage.getItem('list')));
+    console.log(toDoTasks);
+  }
```
[the remark this answers is not in the document]
Comment (BregornOriginal) [reply src/index.js]: Hi, @Gambit142  I missed them! Deleted!

File: src/index.js [hunk 2]
```diff
@@ -1,65 +1,122 @@
...
+    this.completed = completed;
+    this.index = index;
   }
+}
+
+let toDoTasks = [];
+
+const $listContainer = document.getElementById('list-container');
+const $textNewTask = document.getElementById('textNewTask');
+const $addButton = document.querySelector('.add-button');
```
[the remark this answers is not in the document]
Comment (BregornOriginal) [reply src/index.js]: Ok, Added. Now I continue whit the last part!

**Review by AshakaE — APPROVED** [review]
Hi @BregornOriginal , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**